config/checkstyle_non_main_files_checks.xml

```diff
@@ -11,7 +11,7 @@
 
     <!-- Suppresions for resources -->
     <module name="SuppressionFilter">
-        <property name="file" value="${project.basedir}/config/checkstyle_non_main_files_suppressions.xml"/>
+        <property name="file" value="config/checkstyle_non_main_files_suppressions.xml"/>
```

Comment:
why can't we keep `${project.basedir}`?
The other one, `checkstyle_checks.xml`, has all its. If we want to run this config on external projects like sevntu, we probably need it.